ExtraDry/ExtraDry.Blazor/Models/IndicatorSize.cs

```diff
@@ -16,5 +16,9 @@ public enum IndicatorSize {
     /// <summary>
     /// Displays as a large indicator icon
     /// </summary>
-    Large
+    Large,
+    /// <summary>
+    /// Displays no icon
```

Comment:
Consistent documentation with other enum, should be "Displays no indicator icon"